src/test/resources/simple/not-allowed-url-bag/fetch.txt

```diff
@@ -0,0 +1 @@
+ftp://dans.knaw.nl/nl/over/organisatie-beleid/informatiemateriaal/Software_Sustainability_DANS_NLeSC_2016.pdf  555292  data/brochure/Software_Sustainability_DANS_NLeSC_2016.pdf
```

Comment:
Here we actually expect that this url will not be accepted.
But I use` https://dans.knaw.nl/nl/over/organisatie-beleid/informatiemateriaal/Software_Sustainability_DANS_NLeSC_2016.pdf` in valid bags. I am aware that this url might not remain forever, but I expect that it will not disappear in the near future.